entity-service-impl/src/main/java/org/hypertrace/entity/data/service/EntityNormalizer.java

```diff
@@ -100,7 +103,11 @@ private boolean isV2Type(String entityType) {
     return this.entityTypeV2Client
         .get(entityType)
         .map(unused -> true)
-        .onErrorReturnItem(false)
+        .onErrorResumeNext(
+            throwable ->
+                throwable instanceof NoSuchElementException
```

Comment:
Changing the thrown error into a grpc NOT_EXISTS exception I thought would make sense and be consistent with other places.